README.md

```diff
@@ -1,5 +1,5 @@
 [![CircleCI - Build Status](https://img.shields.io/circleci/build/github/golang-migrate/migrate/master)](https://circleci.com/gh/golang-migrate/migrate)
-[![GoDoc](https://godoc.org/github.com/golang-migrate/migrate?status.svg)](https://godoc.org/github.com/golang-migrate/migrate)
+[![GoDoc](https://godoc.org/github.com/golang-migrate/migrate?status.svg)](https://pkg.go.dev/github.com/golang-migrate/migrate/v4)
```

Comment:
Also update the [image link](https://pkg.go.dev/badge/github.com/golang-migrate/migrate)

Reply:
Ah, didn't realize they now exposed those directly... update! 

I dropped the `?status.svg` as it appears to work fine without that, but if you want me to put it back let me know.